src/Zend/Server/Reflection/Node.php

```diff
@@ -52,10 +52,10 @@ class Zend_Server_Reflection_Node
      * Constructor
      *
      * @param mixed $value
-     * @param Zend_Server_Reflection_Node $parent Optional
+     * @param Zend_Server_Reflection_Node|null $parent Optional
```

Comment:
Did you leave this line for the "Optional" comment only ?

Reply:
Yeah, I usually remove the docblock definitions for an argument if it matches the native type hint unless there's some additional information in the docblock, like in this case (this extra comment isn't probably worth keeping the docblock for, but it's not really a big deal).